charts/gitops-runtime/README.md

```diff
@@ -27,7 +27,7 @@ We have created a helper utility to resolve this issue:
 The utility is packaged in a container image. Below are instructions on executing the utility using Docker:
 
 ```
-docker run -v <output_dir>:/output quay.io/codefresh/gitops-runtime-private-registry-utils:0.8.1 <local_registry>
+docker run -v <output_dir>:/output quay.io/codefresh/gitops-runtime-private-registry-utils:0.0.0 <local_registry>
```

Comment:
Not a typo?

Reply:
i just ran `helm-docs.sh`.
maybe i shouldn't have done it? is it all handled in the github action workflows on release?

Reply:
Not sure why it did that. But yeah, it probably will be overridden during the release process.